docs/codeql/codeql-language-guides/specifying-additional-remote-flow-sources-for-javascript.rst

```diff
@@ -3,13 +3,16 @@
 Specifying additional remote flow sources for JavaScript
 ========================================================
 
-You can model potential sources of untrusted user input in your code without making changes to the CodeQL standard library by specifying extra remote flow sources in an external file.
-
 .. pull-quote::
 
-   Note
+   Deprecation Notice
+
+   Specifying remote flow sources with the JSON format described here is soon to be deprecated
+   and will be removed in the future.
 
-   Specifying remote flow sources in external files is currently in beta and subject to change.
+   See :ref:`Customizing Library Models for JavaScript <customizing-library-models-for-javascript>`.
```

Comment:
Can we remove this for now since we don't want customization to be linked to publicly?